CONTRIBUTING.md

```diff
@@ -48,6 +48,26 @@ If several `It` sections are similar, consider refactoring them in a `DescribeTa
 - [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide#contributing)
 - [Contributor Cheat Sheet](https://git.k8s.io/community/contributors/guide/contributor-cheatsheet) - Common resources for existing developers
 
+## Managing releases
+
+### OperatorHub.io
+
+After a new version is released, it must be added manually to the
+[k8s-operatorhub/community-operators](https://github.com/k8s-operatorhub/community-operators) repo to be available on
+OperatorHub.io.
+
+First, generate all manifests:
+
+```shell
+make operator-hub-release IMAGE_TAG=<RELEASE_TAG> VERSION=<VERSION_NUMBER>
+```
+
+This will create two directories, `kmm-operatorhub-<VERSION>` and `kmm-hub-operatorhub-<VERSION>`.
+
+Then, use the contents of those directories to create new versions in
+[k8s-operatorhub/community-operators](https://github.com/k8s-operatorhub/community-operators).
+Only one operator should be affected per pull request; create separate PRs if you want to update both KMM bundles.
```

Comment:
Why is that? Why can't we make a change to both KMM and KMM-hub in a single PR?
Adding a new feature to hub&spoke will probably modify both KMM and KMM-hub.

Reply:
The checklist when creating a new PR mentions this limitation.

Reply:
You mean PRs to https://github.com/k8s-operatorhub/community-operators?

Reply:
The checklist is the default PR template. See https://github.com/k8s-operatorhub/community-operators/pull/2858 for an example